src/components/Nav/CategoryTab.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import './CategoryTab.scss';
+
+const CategoryTab = () => {
+  const [category, setCategory] = useState([]);
+  const [activeTab, setActiveTab] = useState(0);
+
+  useEffect(() => {
+    fetch('/data/categoryMock.json', {
+      method: 'GET',
+    })
+      .then(response => response.json())
+      .then(data => {
+        setCategory(data);
+      });
+  }, []);
+
+  const clickHandler = id => {
+    setActiveTab(id);
+  };
+
+  return (
+    <div className="categoryTab">
+      <div className="tabs">
+        <ul className="tabList">
+          {category.map((el, i) => {
```

Comment:
통상적으로 콜백함수의 매개변수의 el, i 라고 많이 사용하지만, 어떤 데이터가 들어오는지 알수있는 매개면수로 네이밍 해주시는 게 가독성이 좋아질 것 같아요!
네이밍에 고민이 되신다면, 매개변수 자리에서 구조분해 할당을 해주는 것도 방법입니다!
```suggestion
          {category.map(({id, name}, i) => {
```

Reply:
리뷰 반영하여 수정하였습니다!